### DISCUSSION TITLE
Mailer Error: SMTP Error: Could not connect to SMTP host.

### DISCUSSION THREAD

**Discussion by Aleph-Design** [PHPMailer/PHPMailer#2577] [General]
When using PHPMailer from my localhost all is well as I use:
  $mail->SMTPSecure = PHPMailer::ENCRYPTION_SMTPS; 
  $mail->Port       = 465; 
Using:
  $mail->SMTPSecure = PHPMailer::ENCRYPTION_STARTTLS;
  $mail->Port       = 587; 
I get this error:
  Mailer Error: SMTP Error: Could not connect to SMTP host.

This is most likely because my MAMP server doesn't support TLS, but
wouldn't an automatic fall back on SSL be nice?

**Comment by Synchro** [comment]
Not really. You're expected to configure your script to work with your server. The only thing that PHPMailer does automatically is to auto-enable encryption if a server advertises it, however even that had to have [an option to disable it](http://phpmailer.github.io/PHPMailer/classes/PHPMailer-PHPMailer-PHPMailer.html#property_SMTPAutoTLS) because some people run broken mail servers...
